.vscode/settings.json

```diff
@@ -1,6 +1,6 @@
 {
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true,
+    "source.fixAll.eslint": "explicit",
```

Comment:
Why?

Reply:
no idea, tento subor som neriesila. Teraz som menila na true, ale to nejde commitnut.